server/src/main/scala/org/scassandra/server/priming/json/ResultJsonRepresentation.scala

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
If we go this route there's a few other json/webapi specific classes that'd be worth moving.